code/game/objects/items/devices/PDA/PDA.dm

```diff
@@ -12,8 +12,7 @@ var/global/list/obj/item/device/pda/PDAs = list()
 	w_class = 2.0
 	slot_flags = SLOT_ID | SLOT_BELT
 	sprite_sheets = list("Resomi" = 'icons/mob/species/resomi/id.dmi')
-	//offset_light = 1
-	//diona_restricted_light = 1//Light emitted by this object or creature has limited interaction with diona
+	uv_intensity = 15

```

Comment:
I would like to get Hurtz' and @skull132's opinion on PDAs emitting *any* UV light.